posts/aws-summit-2018nyc.md

```diff
@@ -0,0 +1,38 @@
+
+## Databases 
+Amazon is pushing its managed Database services hard. There were recurring workshops on [Aurora](https://aws.amazon.com/rds/aurora/), Amazon's fully managed solution intended to replace your MySql or PostgreSQL. This includes the release of Aurora Serverless, which can autoscale up or down depending on the needs of a severless application. If you are not looking to go all-in on AWS there are other options as well. MongoDB was present pitching [Atlas](https://www.mongodb.com/cloud/atlas), their fully-managed NoSQL database in the cloud for AWS, GCP, and Azure. 
+
+Measuring the true cost of a fully-managed database vs a cloud hosted solution vs an on-premise solution is not a trivial task and a lot of factors have to be considered. But for business solutions that don't want to worry about the headache of patching, upgrading, scaling and fine-tuning a solution, a managed solution offers many nice features.
+
+## Serverless Architectures
+Full stack developers and Solution Architects have a myriad of options at their disposal. In addition to all the backend storage options mentioned above,[serverless architecture patterns](https://blog.ippon.tech/going-serverless/) are transforming the way Internet solutions can be architected in the cloud. 
+
+Applications that are data intensive can use Kinesis Streams to ingest and process massive amounts of data (a presentation from the world's most popular mobile game revealed that the developers are ingesting and processing petabytes of data each day). 
+
+While Lambdas are not new, the way they are being integrated into AWS is. Your EC2 instances might go the way of the Dodo and you'll probably see your monthly bill go down dramatically.
+
+But if they don't cut it Amazon is introducing new high performance instance types including the [Z1d instances](https://aws.amazon.com/ec2/instance-types/z1d/) are speced out at Ferrari levels with 4.0 GHz clock speeds, SSD drives, 48 CPUs and over 380 gigs of RAM. 
+
+## Machine Learning
+If there was one topic that inspired the most excitement and had the biggest WTF-factor it would have to be the Machine Learning/AI technologies that are being offered. With frameworks in place for managing training data sets and gathering insights from them. The algorithms for doing this type of processing have been around for decades, but the ability to process them at scale has always been a limiting factor until recently. Amazon offers [SageMaker](https://aws.amazon.com/sagemaker/) with coming support for [TensorFlow](https://www.tensorflow.org/) and other ML Frameworks to leverage this capability for an organization.
```

Comment:
"With frameworks in place for managing training data sets and gathering insights from them." is that a continuation of the previous sentence or is it supposed to stand on it's own? Seems a little confusing by itself.